### PR TITLE
Adds stage separation to vara-cs status

### PR DESCRIPTION
**Pull request by vulder**
resolves se-passau/VaRA#425

### CODE REVIEW

**Comment by vulder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- varats/paper/case_study.py  3
- varats/driver.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/VaRA/VaRA-Tool-Suite/pullRequest?prid=3715243&bid=13081610)

**Comment by vulder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- varats/paper/case_study.py  3
- varats/driver.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/VaRA/VaRA-Tool-Suite/pullRequest?prid=3715243&bid=13081610)

**Comment by vulder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- varats/paper/case_study.py  3
- varats/driver.py  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/VaRA/VaRA-Tool-Suite/pullRequest?prid=3715243&bid=13081610)

**Comment by vulder** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- varats/driver.py  2
- varats/paper/case_study.py  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/VaRA/VaRA-Tool-Suite/pullRequest?prid=3715243&bid=13081610)

File: varats/driver.py [hunk 1]
```diff
@@ -438,8 +443,12 @@ def add_common_args(sub_parser):
             parser.error(
                 "At least one argument of: --short, --list-revs can be used.")
 
+        if args['short'] and args['ws']:
+            parser.error(
+                "At least one argument of: --short, --ws can be used.")
```
Comment (julianbreiteneicher) [inline varats/driver.py]:
```suggestion
                "At most one argument of: --short, --ws can be used.")
```

**Review by julianbreiteneicher — APPROVED** [review]
One typo, otherwise looks good

**Pull request merged**